sicp-pict/main.rkt

```diff
@@ -361,26 +382,23 @@
 ;;;
 
 (provide painter/c
-         ;
-         with-transformation
-         with-frame
-         with-pen
-         with-brush
          ;
```

Comment:
Why are these not exported?

Reply:
They are exported in other position (under Syntactic Sugar).